idd/Energy+.idd.in

```diff
@@ -19678,11 +19678,11 @@ People,
        \minimum 0.0
        \maximum 1.0
   A5 , \field Activity Level Schedule Name
-       \required-field
        \note Note that W has to be converted to mets in TC routine
        \type object-list
        \note units in schedule are W/person
        \object-list ScheduleNames
+       \note optional (only required for runs of thermal comfort models: Fanger, Pierce and KSU)
```

Comment:
Not so.  Activity level is used in the zone heat balance.  This should remain required.